release-notes.md

```diff
@@ -10,6 +10,10 @@ The following updates were made in this release:
 * Local notifications now use the `UserNotifications` framework.
 * Fixed a crash that might happen when the **Click-Through** or the **Cancel** button in the Full-screen message was clicked more than once.
 
+**Android Analytics 1.2.1**
+
+* Fixed a bug where the Analytics database was being modified by multiple threads on startup.
```

Comment:
Please reword as follows:
" Fixed a bug where, on start up, the Analytics database was being modified by multiple threads."